src/windows/functions.cpp

```diff
@@ -393,6 +393,12 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterValid(TCHAR driveLetter) {
+  int driveLetterIndex = static_cast<int>(driveLetter) - static_cast<int>('A');
```

Comment:
> Is there any advantage of avoiding the cast?

Nah, probably not, just a stylistic thing.

> I think it should work fine, but I'll test that out.

Make sure you also test (if you can) having a network drive as e.g. E: and then try ejecting a removable drive at G: